src/java/org/apache/cassandra/service/accord/AccordCallback.java

```diff
@@ -32,6 +32,7 @@
 import org.apache.cassandra.net.Message;
 import org.apache.cassandra.net.RequestCallback;
 
+// TODO (review): Why is this unused now, should it be removed? Was it a mistake?
```

Comment:
@belliottsmith can you comment here?  Looks like you migrated all messages off C* callbacks...

Reply:
Yeah, looks like I forgot to delete this. We use `RequestCallbacks` now and operate on native Accord callbacks, which have better expiration guarantees and also handle reporting of responses that are behind their predicted schedule so that we can try alternative replicas.

We should consider eventually replacing the regular Cassandra callbacks with the Accord ones.

Reply:
I'll remove the file